src/pks/transport/transport_ats_ti.cc

```diff
@@ -130,31 +137,35 @@ Transport_ATS::AddAdvection_FirstOrderUpwind_(double t_old,
 
 
 void
-Transport_ATS::AddAdvection_SecondOrderUpwind_(double t_old,
-        double t_new,
-        const Epetra_MultiVector& tcc,
-        Epetra_MultiVector& cq_flux)
+Transport_ATS::AddAdvection_SecondOrderUpwind_(
+  double t_old, double t_new,  // old and new time
+  const Epetra_MultiVector& tcc,  // concentration in a cell at old time [mols C / mols H2O]
+  Epetra_MultiVector& c_qty,  // component quantity in a cell [mols C]
+  Epetra_MultiVector& cq_flux)  // component flux across a face [mols C / s]
 {
   for (int i = 0; i != num_aqueous_; ++i) {
-    AddAdvection_SecondOrderUpwind_(t_old, t_new, *tcc(i), *cq_flux(i), i);
+    AddAdvection_SecondOrderUpwind_(t_old, t_new, *tcc(i), *c_qty(i), *cq_flux(i), i);
   }
 }
 
 
 //
 // Given C_old, this computes div Cq using a second order limiter
 //
-// DEV NOTE: requires that tcc is ghosted and scattered, while cq_flux is
+// DEV NOTE: requires that tcc is ghosted and scattered, while c_qty is
 // owned.
 void
-Transport_ATS::AddAdvection_SecondOrderUpwind_(double t_old,
-        double t_new,
-        const Epetra_Vector& tcc,
-        Epetra_Vector& cq_flux,
-        int component)
+Transport_ATS::AddAdvection_SecondOrderUpwind_(
+  double t_old, double t_new,
+  const Epetra_Vector& tcc,
+  Epetra_Vector& c_qty,
+  Epetra_Vector& cq_flux,
+  int component)
 {
-  double dt = t_new - t_old;
+  double dt = t_new - t_old;  // time step [s]
```

Comment:
please spell this "timestep".  We made this uniform across the codebase for easier grepping a bit ago, and while it's not particularly important in a comment, I'd rather keep it consistent everywhere.